minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java

```diff
@@ -45,6 +47,7 @@
 /**
  * Local Tachyon cluster.
  */
+@NotThreadSafe
```

Comment:
No convention for interfaces yet. 

For abstract classes, the convention is that it is the contract the implementations should meet (e.g. NotThreadSafe means implementation can be NotThreadSafe or ThreadSafe, while ThreadSafe means implementations should be ThreadSafe).